.github/workflows/ci.yml

```diff
@@ -449,15 +450,14 @@ jobs:
               ./build_tools/scripts/check_vulkan.sh
               build_tools/cmake/run_tf_tests.sh ${BUILD_DIR}"
 
-  ############################# Model Coverage #################################
-  # Jobs that test IREE performance on a set of models.
+  ######################## Community Model Coverage ############################
+  # Jobs that test IREE behavior on a set of models.
   ##############################################################################
   test_shark_model_suite:
     needs: [setup, build_all, build_tf_integrations]
-    if: needs.setup.outputs.should-run == 'true'
+    if: needs.setup.outputs.should-run == 'true' && needs.setup.outputs.ci-stage == 'postsubmit'
```

Comment:
https://github.com/nod-ai/SHARK/issues/373#issuecomment-1261203723

> I think disabling pending a fix makes sense. It's not giving us any coverage as is. But please also merge this change :-) I'd just put a "false" in the if condition here so we preserve the configuration

@GMNGeoffrey want `false` in this condition in this PR? Or start with postsubmit but expected to fail (takes just 3m 5s, so not wasting a ton of resources...)

Reply:
PR would probably be slightly better just for history. Then it can be reverted with just a `git revert`. But fine in this PR too :-)